test/iterables/avg_test.dart

```diff
@@ -0,0 +1,16 @@
+library quiver.iterables.avg_test;
+
+import 'package:test/test.dart';
+import 'package:quiver/iterables.dart';
+
+void main() {
+  group('avg', () {
+    test('should return the average of elements', () {
+      expect(avg([2, 5, 1, 4]), 3);
```

Comment:
Test passes, but I'm not sure if it's a good idea to use strict equality with doubles in Dart. Is there any better/idiomatic way?